installer.lua

```diff
@@ -398,8 +397,6 @@ local function install()
                     "The code has already been copied to your clipboard:",
                     ">lua",
                     " local rocks_config = {",
-                    '     rocks_path = vim.env.HOME .. "' .. install_path_rel .. '",',
```

Comment:
question: was this removal by mistake? This line is still used below 👀

Reply:
Yep, that was unintended